examples/dmd/heat_conduction_otf.cpp

```diff
@@ -0,0 +1,711 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2013-2023, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+
+//                       libROM MFEM Example: Heat_Conduction (adapted from ex16p.cpp)
+//
+// Compile with: make heat_conduction
+//
+// =================================================================================
+//
+// Sample runs and results for DMD:
```

Comment:
This sample runs must be replaced with heat_conduction_otf examples.

Reply:
Also document what `otf` means.

Reply:
Fixed in c0c76ad. I just changed the name to `heat_conduction_incdmd` for clarity.

Reply:
Updated in 3459c78.